### ISSUE TITLE
Error in vignette: Unknown variable: agegp+

### ISSUE THREAD

**Issue by dmenne** [nbarrowman/vtree#33]
```
 vtree(esoph,"agegp=75+",sameline=TRUE,cdigits=0,
+       summary=c("ncases \ncases=%sum%%leafonly%","ncontrols  controls=%sum%%leafonly%"))
Error in vtree(esoph, "agegp=75+", sameline = TRUE, cdigits = 0, summary = c("ncases \ncases=%sum%%leafonly%",  : 
  Unknown variable: agegp+
```

**Comment by nbarrowman** [comment]
Sorry I missed this issue until now. I will fix the error with `agegp=75+` and release this soon in a new version of vtree.

I wonder if the `Error in justpct(y, digits = digits, thousands = thousands)` has been fixed in version 5.6.4 of vtree?

Thanks,
Nick

**Comment by dmenne** [comment]
Works fine with 5.6.4! Thanks, Nick.
